alacritty_terminal/src/term/mod.rs

```diff
@@ -2429,6 +2722,288 @@ mod tests {
         assert_eq!(term.grid.cursor.point, Point::new(Line(4), Column(0)));
     }
 
+    #[test]
```

Comment:
Haven't really reviewed the tests, maybe one comment I could leave about them is that they do look kinda long and complicated. So that's somewhat discouraging. Usually prefer things to be small and easy to understand in case someone needs to change things in the future.

Reply:
They just cover **all** functions which are damaging terminal in one way or another. So those tests are made to hit coverage, and not be short. But they are really simple if you look into them. They basically do `damage -> check result -> reset` and repeat until we cover all functions affected.

Reply:
Yeah I saw that for some of them. Still somewhat annoying to quickly determine if they're actually doing what they're supposed to.